src/DssSpell.t.sol

```diff
@@ -147,7 +169,7 @@ contract DssSpellTest is DSTest, DSMath {
     }
 
     // Previous spell; supply if there is a need to test prior to its cast() function being called on mainnet.
-    SpellLike constant PREV_SPELL = SpellLike(0x8D602692eE4b5f0ec33A22fe6547822377FDCc4c);
+    SpellLike constant PREV_SPELL = SpellLike(0x823Ac093BC6C5D1cC48739d574aC0a8D09ffC565);
```

Comment:
This has already been `cast` on mainnet, I believe, so arguably we could just set this to 0. OTOH, the test is set up so that this doesn't matter. Just bringing it up in case we want to think about how we'd like to handle this going forward (always put in previous spell, or only put it in if it actually needs to be `cast`).

Reply:
It wasn't cast yet, will be done tomorrow. I guess that's why the comment was resolved, but just in case wanted to confirm it is necessary in this case.